hhs_oauth_server/middleware.py

```diff
@@ -0,0 +1,11 @@
+from django.utils.deprecation import MiddlewareMixin
+from rest_framework.exceptions import APIException
+from django.http import HttpResponse
+
+
+# Mixin for compatibility with Django <1.10
+class HandleAPIExceptionMiddleware(MiddlewareMixin):
+    def process_exception(self, request, exception):
+        if isinstance(exception, APIException):
+            return HttpResponse(content=exception.detail,
+                                status=exception.get_codes())
```

Comment:
The get_codes function isn't meant for HTTP status codes, and isn't guaranteed to return a valid status code. The examples in their docs are mostly dictionaries. I think you've implemented this in such a way that this works here, but it's not really how its intended to be used.

Reply:
OK I'll subclass APIException so we can make better use of the `status_code` attribute.